_inc/client/components/module-settings/connect-module-options.jsx

```diff
@@ -38,7 +38,12 @@ export function connectModuleOptions( Component ) {
 	return connect(
 		( state, ownProps ) => {
 			return {
-				validValues: ( option_name ) => getModuleOptionValidValues( state, ownProps.module.module, option_name ),
+				validValues: ( option_name, module_slug = '' ) => {
+					if ( 'object' === typeof ownProps.module && 'undefined' !== typeof ownProps.module.module ) {
```

Comment:
Possible candidate for using lodash's `get()` here instead of checking parent members.